tests/unit/tools/test_google_docs.py

```diff
@@ -0,0 +1,39 @@
+from griptape.tools import GoogleDocsClient
+from unittest.mock import patch
+from google.auth.exceptions import MalformedError
+import pytest
+
+
+class TestGoogleDocsClient:
+
+    @pytest.fixture
+    def mock_docs_client(self):
+        return GoogleDocsClient(
+            owner_email="tony@griptape.ai",
+            service_account_credentials={}
+        )
+
+    def test_create_google_doc(self, mock_docs_client):
+        file_name = "test_document"
+        assert "Error creating Google Doc" in mock_docs_client.create_google_doc(file_name=file_name).value
```

Comment:
Store the result first, and then assert the string on a new line.

Reply:
Done!